src/main/java/org/apache/maven/plugins/dependency/utils/CopyUtil.java

```diff
@@ -55,20 +55,27 @@ public CopyUtil(BuildContext buildContext) {
      *
      * @param sourceArtifact the artifact (file) to copy
      * @param destination file name of destination file
+     * @param silent if true, use debug log level, otherwise info for logging
```

Comment:
This feels wrong. Logging should be controlled by log levels, not Maven parameters